docs/OPERATOR_VIEW.md

```diff
@@ -15,7 +15,7 @@ Each BPDM application is a SpringBoot Kotlin software project managed by Maven.
 To run the application first you need to install it from the parent pom. For that go to the root folder and then there are two options to run the project:
 
 1. `mvn clean install -DskipTests` - This will install all the dependencies without need for test execution.
-2. Install missing docker images used by the bridge-dummy tests `docker compose -f docker-compose.build.yml build`. After that run `mvn clean install`.
+2. Install missing docker images used by the bridge-dummy tests `docker compose -f docker-compose.yml build`. After that run `mvn clean install`.
```

Comment:
True there is no build yaml anymore but we also have no Bridge Dummy anymore very recently. So actually we should reformulate the whole paragraph. Currently, everything can be easily installed by just a mvn clean install. No need to prepare anything before installing.

However instead we should mention to execute the docker compose file in order to run the dependencies needed before a user starts the installed programm with `mvn spring-boot:run`